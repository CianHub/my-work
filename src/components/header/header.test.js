// contact.contact.test.js
import React from "react";
import Header from "./header";
import renderer from "react-test-renderer";

test("Icon is rendered", () => {
  const component = renderer.create(
    <Header text="I'm the text" textId="testID" hrId="testHR" />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("Icon is rendered with white coloring", () => {
  const component = renderer.create(
    <Header text="At Your Service" textId="aboutText" hrId="aboutHr" />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
