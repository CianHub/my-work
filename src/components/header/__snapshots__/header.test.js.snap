// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon is rendered 1`] = `
<div
  className="container header"
>
  <div
    className="row"
  >
    <div
      className="col-lg-12 text-center"
    >
      <h2
        className="section-heading"
        id="testID"
      >
        I'm the text
      </h2>
      <hr
        className="my-4"
        id="testHR"
      />
    </div>
  </div>
</div>
`;

exports[`Icon is rendered with white coloring 1`] = `
<div
  className="container header"
>
  <div
    className="row"
  >
    <div
      className="col-lg-12 text-center"
    >
      <h2
        className="section-heading"
        id="aboutText"
      >
        At Your Service
      </h2>
      <hr
        className="my-4"
        id="aboutHr"
      />
    </div>
  </div>
</div>
`;
